src/Components/SearchBar.jsx

```diff
@@ -26,8 +29,8 @@ export default function SearchBar({onClickMenuCb, showNavPanel}) {
   const [inputText, setInputText] = useState('')
   const onInputChange = (event) => setInputText(event.target.value)
   const searchInputRef = useRef(null)
-  const classes = useStyles()
-
+  const inputWidth = Number(inputText.length)*11
+  const classes = useStyles({width: inputWidth})
```

Comment:
I feel like the 11 should be derived from the font size, as an 'em' property.  Either way, more readable to have the 11 close to the usage location, so I think 'width' here should instead be inputTextLength and then the calculation in the useStyles func.

Reply:
yes that makes sense